_irc/index.md

```diff
@@ -357,4 +346,14 @@ these issues and overhaul the notification system, so we do not recommend
 implementing this spec.
 
 
+## [STARTTLS]({{site.baseurl}}/specs/extensions/tls-3.1.html)
+
+STARTTLS allows clients to upgrade their plaintext connections to use TLS
+encryption. It is recommended that clients instead implement STS support.
```

Comment:
This is a bit of a crappy explanation, and should be expanded with more reasoning.